src/newUi.scss

```diff
@@ -1128,17 +1128,19 @@ nav {
   border-bottom: none !important;
 }
 
+.fs-14 {
+  font-size: 14px !important;
+}
+
 .table-details td {
-  color: var(--purple-text);
-  padding: 20px 10px;
+  padding: 20px 0px;
```

Comment:
Removed the padding from the lists. It's not according to Ana's design, but I personally think it looks better.

Original
![image](https://github.com/user-attachments/assets/a91424dd-89ec-4326-945b-b3612612a8ce)


New
![image](https://github.com/user-attachments/assets/a3804250-27b4-4b7b-a571-e88ee70f3313)